src/test/kotlin/de/bund/digitalservice/useid/widget/WidgetControllerIntegrationTest.kt

```diff
@@ -139,9 +137,7 @@ class WidgetControllerIntegrationTest(@Autowired val webTestClient: WebTestClien
         assertThat(containerFallback, hasValidFallbackClassName)
 
         val errorTitle = parsedResponseBody.getElementsByClass("error_title").text()
-        val hasValidErrorTitle = containsString(widgetProperties.errorView.fallback.localization.errorTitle)
-
-        assertThat(errorTitle, hasValidErrorTitle)
+        assertThat(errorTitle, containsString("Es ist ein Fehler aufgetreten."))
```

Comment:
same here. 
How is this going to work with a different locale or do the tests always use the default locale? 
I would try and fetch the asserted value directly from the .properties file, if that is possible.

Reply:
See comment above.